packages/pipeline-editor/style/index.css

```diff
@@ -635,3 +635,8 @@ input.elyra-Dialog-checkbox.jp-mod-styled {
 .toolbar-overflow-menu-item button:hover {
   background-color: var(--jp-border-color1);
 }
+
+/* fix font ligatures */
+.lsp-completer-theme-vscode {
```

Comment:
What is the relation between this and vscode? If none, could we remove vscode from the css tag name.

Reply:
This isn’t from us, we are overriding css from here I believe: https://github.com/krassowski/jupyterlab-lsp/blob/master/packages/theme-vscode/style/completer.css

Reply:
Not sure we can rename the class as we don't have any reference to it in Elyra or any of our pipeline related repositories.
@bourdakos1 do you know if this css class introduced by carbon-components?

Reply:
I think it was introduced from the project linked above ^

Reply:
oh whoops only now I saw your first comment @bourdakos1 , this page wasn't refreshed when I saw Luciano's comment 😅

Reply:
could we also create a pr for jupyterlab-lsp to fix the issue there?

Reply:
@lresende I think this is more of a bandaid fix, because I believe the ligatures work fine with only the lsp extension installed?

Reply:
Correct. I've tested it out again with the the lsp extension only installed and it works fine.